### PR TITLE
new instruments added to sensor bulk load

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added new CTDMO-S sensors (converted for MAB) and PHSEN-Gs.  Also added missing info for the existing PHSEN-G/H sensors.  Confirm via WHOI_Asset_Tracking Ss

### CODE REVIEW

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -2442,9 +2452,18 @@ CGINS-PHSENF-P0180,A01713,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Su
...
+CGINS-PHSENG-02080,A04410,Sensor,0,Seawater pH: PHSEN Series G,PHSEN SERIES G,Sea-Bird Electronics,Deep SeapHOx (350 m),721-2080,,20240802,45000,,Woods Hole Oceanographic Institution
+CGINS-PHSENG-02081,A04409,Sensor,0,Seawater pH: PHSEN Series G,PHSEN SERIES G,Sea-Bird Electronics,Deep SeapHOx (350 m),721-2081,,20240719,45000,,Woods Hole Oceanographic Institution
+CGINS-PHSENG-02082,A04411,Sensor,0,Seawater pH: PHSEN Series G,PHSEN SERIES G,Sea-Bird Electronics,Deep SeapHOx (350 m),721-2082,,20240802,45000,,Woods Hole Oceanographic Institution
+CGINS-PHSENG-02083,A04412,Sensor,0,Seawater pH: PHSEN Series G,PHSEN SERIES G,Sea-Bird Electronics,Deep SeapHOx (350 m),721-2083,,20240802,45000,,Woods Hole Oceanographic Institution
+CGINS-PHSENH-02064,A04400,Sensor,0,Seawater pH: PHSEN Series H,PHSEN SERIES H,Sea-Bird Electronics,Deep SeapHOx (2000 m),721-2064,,20240502,45000,,Woods Hole Oceanographic Institution
+CGINS-PHSENH-02065,A04401,Sensor,0,Seawater pH: PHSEN Series H,PHSEN SERIES H,Sea-Bird Electronics,Deep SeapHOx (2000 m),721-2065,,20240502,45000,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:2466]: looks good

**Review by cooleyky — APPROVED** [review]
looks good!

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -1046,8 +1046,18 @@ CGINS-CTDMOS-12337,A02068,Sensor,0,CTD Mooring (Inductive): CTDMO Series S,CTDMO
...
+CGINS-CTDMOS-13615,A01577,Sensor,0,CTD Mooring (Inductive): CTDMO Series S,CTDMO SERIES S,Sea-Bird Electronics,SBE 37-IM,37-13615,,20150701,7000,converted from CTDMO-Q to -S for Pioneer MAB,Woods Hole Oceanographic Institution
+CGINS-CTDMOS-13618,A01578,Sensor,0,CTD Mooring (Inductive): CTDMO Series S,CTDMO SERIES S,Sea-Bird Electronics,SBE 37-IM,37-13618,,20150701,7000,converted from CTDMO-Q to -S for Pioneer MAB,Woods Hole Oceanographic Institution
+CGINS-CTDMOS-13619,A01579,Sensor,0,CTD Mooring (Inductive): CTDMO Series S,CTDMO SERIES S,Sea-Bird Electronics,SBE 37-IM,37-13619,,20150701,7000,converted from CTDMO-Q to -S for Pioneer MAB,Woods Hole Oceanographic Institution
+CGINS-CTDMOS-13625,A01588,Sensor,0,CTD Mooring (Inductive): CTDMO Series S,CTDMO SERIES S,Sea-Bird Electronics,SBE 37-IM,37-13625,,20150701,7000,converted from CTDMO-G to -S for Pioneer MAB,Woods Hole Oceanographic Institution
+CGINS-CTDMOS-13635,A01590,Sensor,0,CTD Mooring (Inductive): CTDMO Series S,CTDMO SERIES S,Sea-Bird Electronics,SBE 37-IM,37-13635,,20150701,7000,converted from CTDMO-G to -S for Pioneer MAB,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:1060]: CTDMOs look good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:2466]: phsens look good

**Review by cdobs — APPROVED** [review]

**Pull request merged**